app/errors.js

```diff
@@ -11,3 +11,9 @@ exports.defaultError = message => internalError(message, exports.DEFAULT_ERROR);
 
 exports.SERVICE_ERROR = 'service_error';
 exports.serviceError = message => internalError(message, exports.SERVICE_ERROR);
+
+exports.UNPROCESSABLE_ENTITY = 'unprocessable_entity';
+exports.unprocessableEntity = message => internalError(message, exports.UNPROCESSABLE_ENTITY);
+
+exports.BAD_REQUEST = 'bad_request';
```

Comment:
Remember to also add the status code in the file `middlewares/errors.js` otherwise, it will be assigned a status code 500.

Reply:
You did not add the status code for `bad_request` in the error catalog in the `middlewares/errors.js` file this causes the request validator to return an error with status 500.